TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/StrafeTest.java

```diff
@@ -29,5 +29,13 @@ public void runOpMode() throws InterruptedException {
         if (isStopRequested()) return;
 
         drive.followTrajectory(trajectory);
+
+        Pose2d poseEstimate = drive.getPoseEstimate();
+        telemetry.addData("x", poseEstimate.getX());
```

Comment:
Would `"finalX"` be slightly more clear? I honestly could go either way.

Reply:
I think that's a great idea.